File: Mi-Primer-Desafio-js-curso-2-Funciones-y-listas-Interactuando-Con-HTML-Y-Java-Script/funciones.py
#Crear una función que muestre "¡Hola, mundo!" en la consola.

def saludar ():
    print("hola como estas")

saludar()

#Crear una función que reciba un nombre como parámetro y muestre "¡Hola, [nombre]!" en la consola.

def saludar (nombre):
    print(f"hola {nombre}")

saludar("felipe")

#Crear una función que reciba un número como parámetro y devuelva el doble de ese número.

def operacion (numero):
    doble = numero * 2
    print(f"el doble de {numero} es {doble}")

operacion(50)

#Crear una función que reciba tres números como parámetros y devuelva su promedio.


def promedio (numero1,numero2,numero3):
    mioperacion = (numero1 + numero2 + numero3)/3
    print(f"el promedio de {numero1},{numero2} y {numero3} es {mioperacion}")

promedio(25,55,30)    

#Crear una función que reciba dos números como parámetros y devuelva el mayor de ellos.

def mayorDeDosNumeros (minumero1,minumero2,):
    if minumero1 > minumero2:
        print(f"el mayor de los números es {minumero1}")
    elif minumero1 < minumero2:
        print(f"el mayor de los números es {minumero2}")
    else:
        print(f"los números son iguales ")    

mayorDeDosNumeros(85,70)

# Crear una función que reciba un número como parámetro y devuelva el resultado de multiplicar ese número por sí mismo.

def multiplicarNumero(elGranNumero):
    multiplicación = elGranNumero * elGranNumero
    print(f"la multiplicación para {elGranNumero} es { multiplicación}")

multiplicarNumero(80)   

File: Mi-Primer-Desafio-js-curso-2-Funciones-y-listas-Interactuando-Con-HTML-Y-Java-Script/funciones.js
//Crear una función que muestre "¡Hola, mundo!" en la consola.

function saludar() {
    console.log("hola mundo")
}
saludar();

//Crear una función que reciba un nombre como parámetro y muestre "¡Hola, [nombre]!" en la consola.

function saludo(nombre) {
    console.log(`Hola ${nombre}`)
}

saludo("felipe");

//Crear una función que reciba un número como parámetro y devuelva el doble de ese número.

function operacion(numero) {
    let doble = numero * 2
    console.log(`el doble de ${numero} es ${doble}`)
    
}

operacion("8");

//Crear una función que reciba tres números como parámetros y devuelva su promedio.

function promedio(numero1,numero2,numero3) {
    let operacionNumeros = (numero1 + numero2 + numero3)/3
    console.log(`el promedio de los números ${numero1},${numero2},y ${numero3} es ${operacionNumeros}`) 

}

promedio(25,55,30);

//Crear una función que reciba dos números como parámetros y devuelva el mayor de ellos.


function mayorDeDosNumeros(numero3,numero4){
    if (numero3 > numero4){
        console.log(`el mayor de los números es ${numero3}`)
    }else if (numero3 < numero4){
        console.log(`el mayor de los números es ${numero4}`)
    }else{
        console.log("los números son iguales")
    }
}

mayorDeDosNumeros(40,40);

//Crear una función que reciba un número como parámetro y devuelva el resultado de multiplicar ese número por sí mismo.

function multiplicacionNumero(miGranNumero) {
    let multiplicación = miGranNumero * miGranNumero
    console.log(`la multiplicación de ${miGranNumero} es ${multiplicación}`)
    
}

multiplicacionNumero(8);
